examples/machine-controller.yaml

```diff
@@ -111,14 +111,43 @@ rules:
   - list
 ---
 apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: endpoints-reader-default
+  namespace: default
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - endpoints
+  verbs:
+  - get
+  - watch
+  - list
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: RoleBinding
 metadata:
-  name: machine-controller
+  name: machine-controller-kube-public
```

Comment:
This does not work, the `RoleBinding` has to be in the same namespace as the `Role`